src/bench/load_external.cpp

```diff
@@ -48,18 +48,29 @@ static void LoadExternalBlockFile(benchmark::Bench& bench)
     // because that first writes a compact size.
     ss << Span{benchmark::data::block413567};
 
-    // Create the test file.
-    {
+    // Ensure the directory exists
+    if (!fs::exists(testing_setup.get()->m_path_root)) {
+    fs::create_directories(testing_setup.get()->m_path_root);
```

Comment:
formatting is completely off - is this still a draft?

Reply:
Forgot to format before committing. Should be ok now :)

Reply:
Did you test it before committing?

Reply:
I compiled and ran `bench_bitcoin`.